DQM/EcalCommon/src/MESetNonObject.cc

```diff
@@ -153,6 +153,8 @@ namespace ecaldqm {
         throw_("Unsupported MonitorElement kind");
     }
 
+    assert(me);  // Dummy check to silence LLVM analyzer warnings
```

Comment:
same here, ` __attribute__((analyzer_noreturn))` should fix this

Reply:
Looks like there is a more standard way - [link](https://en.cppreference.com/w/cpp/language/attributes/noreturn), I'll use it.